spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationConfig.scala

```diff
@@ -15,112 +15,36 @@
-                         rawFormat: String = "xml",
-                         menasCredentialsFactory: MenasCredentialsFactory = InvalidMenasCredentialsFactory,
-                         charset: Option[String] = None,
-                         rowTag: Option[String] = None,
-                         csvDelimiter: Option[String] = None,
-                         csvHeader: Option[Boolean] = Some(false),
-                         csvQuote: Option[String] = None,
-                         csvEscape: Option[String] = None,
-                         cobolOptions: Option[CobolOptions] = None,
-                         fixedWidthTrimValues: Option[Boolean] = Some(false),
-                         performanceMetricsFile: Option[String] = None,
-                         rawPathOverride: Option[String] = None,
-                         folderPrefix: Option[String] = None,
-                         persistStorageLevel: Option[StorageLevel] = None,
-                         failOnInputNotPerSchema: Boolean = false
-                       )
-
-object StdCmdConfig {
-
-  def getCmdLineArguments(args: Array[String]): StdCmdConfig = {
-    val parser = new CmdParser("spark-submit [spark options] StandardizationBundle.jar")
-
-    val optionCmd = parser.parse(args, StdCmdConfig(args))
-    if (optionCmd.isEmpty) {
-      // Wrong arguments provided, the message is already displayed
-      System.exit(1)
-    }
-    optionCmd.get
+import za.co.absa.enceladus.standardization.StandardizationCmdConfig.stepName
+
+case class StandardizationConfig(rawFormat: String = "xml",
+                                 charset: Option[String] = None,
+                                 rowTag: Option[String] = None,
+                                 csvDelimiter: Option[String] = None,
+                                 csvHeader: Option[Boolean] = Some(false),
+                                 csvQuote: Option[String] = None,
+                                 csvEscape: Option[String] = None,
+                                 cobolOptions: Option[CobolOptions] = None,
+                                 fixedWidthTrimValues: Option[Boolean] = Some(false),
+                                 rawPathOverride: Option[String] = None,
+                                 failOnInputNotPerSchema: Boolean = false)
+
+object StandardizationConfig {
+  def getCmdLineArguments(args: Array[String]): StandardizationConfig = {
+    val parser = new CmdParser(s"spark-submit [spark options] ${stepName}Bundle.jar")
+
+    val optionCmd = parser.parse(args, StandardizationConfig())
+    optionCmd.getOrElse(StandardizationConfig())
   }
 
-  private class CmdParser(programName: String) extends OptionParser[StdCmdConfig](programName) {
+  private class CmdParser(programName: String) extends OptionParser[StandardizationConfig](programName) {
+    override def errorOnUnknownArgument: Boolean = false
+    override def reportWarning(msg: String): Unit = {}
```

Comment:
It won't, it will just ignore it. I set it like that because it parses the arguments in 2 steps: Using Parser in JobConfig and then StandardizationConfig. This way it gives an error that a parameter for standardization was found when parsing the JobConfig and vice-versa.  Setting `errorOnUnknownArgument`, will not result in an error when it encounters an unexpected argument and `reportWarning` will make it do nothing when this error comes as a warning.

Reply:
I was asking in a comment a bit up. What is the trade-off? 
Is it only the warning for the extra parameters?

Reply:
It is a very useful thing when users make a typo in a parameter name or when they forget to specify a mandatory option to a parameter.

Maybe we can have a single cmd line parser for 2 jobs and just validate when a parameter is passed to the job that is not intended.
Having a single cmd line parser also might help setting up the job that combines both standardization and conformance.